### PR TITLE
A6 time methods first draft

### PR DESCRIPTION
**Pull request by jakemulf**

### CODE REVIEW

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Time methods
+
+In our previous notebook, we showed how to use conditionals on time series data. In this notebook, we show how to use Deephaven's time methods on time series data.
```
[the remark this answers is not in the document]
Comment (margaretkennedy) [reply demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:3]: Users might jump around. I don't think the summary is necessary.

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+```python
+even_seconds = daily_data.where("secondOfMinute(Timestamp, TZ_NY) % 2 == 0")
+odd_seconds = daily_data.where("secondOfMinute(Timestamp, TZ_NY) % 2 == 1")
+
+even_minutes = daily_data.where("minuteOfHour(Timestamp, TZ_NY) % 2 == 0")
+odd_minutes = daily_data.where("minuteOfHour(Timestamp, TZ_NY) % 2 == 1")
+```
+
+## update
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:29]: I don't really consider where or update time methods. The minuteOfHour etc qualify. Downsampling should probably be in here, too. I'd say some calendar stuff, too, but we never documented that.

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakemulf) [reply demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:3]: Works for me, we can apply this to all the other notebooks

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+time_interval = expressionToNanos("T1S")
+offset = expressionToNanos("T1000S")
+now = currentTime()
+
+daily_data = create_random_table(time_interval, start_time=minus(now, offset))
+```
+
+## where
+
+`where` filters can be applied to time series data. This example shows a few ways to filter your time series data.
```
[the remark this answers is not in the document]
Comment (jakemulf) [reply demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md]: I removed this section based on your other comment, so we don't need to link this

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md (hunk 2 above)
Comment (jakemulf) [reply demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md]: I agree, I added a downsampling example

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+time_interval = expressionToNanos("T1S")
+offset = expressionToNanos("T1000S")
+now = currentTime()
+
+daily_data = create_random_table(time_interval, start_time=minus(now, offset))
+```
+
+## Datetime arithmethic
+
+Deephaven supports datetime arithmetic through methods such as [plus](https://deephaven.io/core/docs/reference/time/datetime/plus/) and [minus](https://deephaven.io/core/docs/reference/time/datetime/minus/).
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:19]:
```suggestion
Deephaven supports DateTime arithmetic through methods such as [plus](https://deephaven.io/core/docs/reference/time/datetime/plus/) and [minus](https://deephaven.io/core/docs/reference/time/datetime/minus/).
```

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+This example shows how to subtract 2 hours from a timestamp
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:21]:
```suggestion
This example shows how to subtract 2 hours from a timestamp:
```

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+two_hours = convertPeriod("T2H")
+
+daily_data = daily_data.update("TimestampTwoHoursBefore = minus(Timestamp, two_hours)")
+```
+
+## Downsampling
+
+Downsampling can be done in Deephaven by deciding how to group your data, and then using an appropriate aggregation on the grouped data.
+
+With time series data, binning methods like [lowerBin](https://deephaven.io/core/docs/reference/time/datetime/lowerBin/) can be used to group by time stamps.
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:35]:
```suggestion
With time series data, binning methods like [lowerBin](https://deephaven.io/core/docs/reference/time/datetime/lowerBin/) can be used to group by timestamps.
```

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+daily_data_binned = daily_data.update("TimestampMinute = lowerBin(Timestamp, nanos_bin)")\
+    .dropColumns("Timestamp")\
+    .aggBy(agg_list, "TimestampMinute")
+```
+
+## as-of joins
+
+[as-of joins](https://deephaven.io/core/docs/reference/table-operations/join/aj/) allow you to join data without exact matches. This works very well with time-series data.
+
+This example shows how to join two time series tables that have different densities in time stamps.
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:58]:
```suggestion
This example shows how to join two time-series tables that have different densities in timestamps.
```

**Review by margaretkennedy — APPROVED** [review]

**Pull request merged**